takwimu/templates/takwimu/_includes/profile/topic_detail.html

```diff
@@ -123,6 +123,10 @@ <h1>{{ topic.value.title }}</h1>
   .topic h5{
     font-size: 25px;
   }
+  .data-topic {
+    padding-top: 72px;
+    padding-bottom: 72px;
+  }
```

Comment:
I'm not convinced by this. The idea is always to write less CSS so I'd prefer the `py-5` class.